main.go

```diff
@@ -186,6 +186,10 @@ func main() {
 		config.WithAutoDetect(ad),
 		config.WithLabelFilters(labelsFilter),
 	)
+	err = cfg.AutoDetect()
```

Comment:
In the future we could autodetect more things like if certain CRDs are available. We honestly should do this for Prometheus too.